### PR TITLE
Lesson 6

### PR DESCRIPTION
**Pull request by gittafia**
Made layout of the top block according to the design

### CODE REVIEW

File: Triphouse-project/style.css [hunk 1]
```diff
@@ -9,6 +9,108 @@ body {
     max-width: 1440px;
 } 
 
+/* header */
+
+header {
+    height: 100vh;
+    background: url(./img/top-section.png)
+    no-repeat;
```
Comment (ghost) [inline Triphouse-project/style.css]: Use 1 line.

File: Triphouse-project/index.html [hunk 2]
```diff
@@ -9,7 +9,40 @@
     <link rel="stylesheet" href="style.css"/>
 </head>
 <body>
```
Comment (ghost) [inline Triphouse-project/index.html:11]: 2021-04-16_00h32_26
Wrap into div and add style {max-width: 1440px, margin: 0px auto}

File: Triphouse-project/style.css (hunk 1 above)
Comment (gittafia) [reply Triphouse-project/style.css]: Fixed.

File: Triphouse-project/index.html (hunk 2 above)
Comment (gittafia) [reply Triphouse-project/index.html:11]: Fixed.

**Pull request merged**